app/src/main/res/values/colors.xml

```diff
@@ -23,6 +23,9 @@
   <color name="black">#000000</color>
   <!-- AUDIO COMPONENT -->
   <color name="audioComponentBackground">@color/oppiaDarkBlue</color>
+  <!-- PROFILE COMPONENT -->
+  <color name="avatarBorder">#4E4E50</color>
+  <color name="chooserBackground">#212121</color>
```

Comment:
suggest renaming this to `profileChooserBackground`

Reply:
+1

Reply:
Updated.